installers/pgo-installer/README.md

```diff
@@ -0,0 +1,5 @@
+# Ansible installer for pgo-installer image
+### Ansible command
+```
+ansible-playbook -i $PGOROOT/ansible/inventory --tags=install main.yml
+```
```

Comment:
This needs to be written for directions that actually pertain to running the `pgo-installer` container

Reply:
This was very quick and dirty as a way to show how the installer would be used for purposes of review. Documentation will be more detailed